viewer/packages/pointclouds/src/styling/shapes/Box.ts

```diff
@@ -0,0 +1,51 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+import { ShapeType } from './IRawShape';
+import { IShape } from './IShape';
+import {
+  AABB,
+  createInvertedRevealTransformationFromCdfTransformation,
+  m4MultiplyV3WithTranslation,
+  Mat4,
+  Vec3
+} from './linalg';
+
+export type RawBox = {
+  type: ShapeType.Box;
+  invMatrix: Mat4;
+};
+
+export class Box implements IShape {
+  readonly invMatrix: Mat4;
+
+  constructor(instanceMatrix: number[], alreadyInverted: boolean = false) {
+    if (alreadyInverted) {
+      this.invMatrix = { data: instanceMatrix.slice() };
+    } else {
+      const mat: Mat4 = { data: instanceMatrix };
+      this.invMatrix = createInvertedRevealTransformationFromCdfTransformation(mat);
+    }
+  }
+
+  computeBoundingBox(): AABB {
+    throw Error('Bounding box not implemented for box primitive');
+  }
```

Comment:
If it is needed and you simply haven't done it yet that is fine. If however, this doesn't make sense for Box, then it breaks the Interface segregation principle.

Reply:
Is needed at a later point, keeping for now